annotation-processors/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+  id("adventure.common-conventions")
+}
+
+dependencies {
+  annotationProcessor(libs.autoService.processor)
+  compileOnlyApi(libs.autoService.annotations)
+  api(libs.jetbrainsAnnotations)
+}
```

```suggestion
}

tasks.withType(AbstractPublishToMaven::class) {
  isEnabled = false
}
```

one last thing, just to make sure we don't publish this module externally